aio.api.github/aio/api/github/abstract/tag.py

```diff
@@ -0,0 +1,26 @@
+
+import abstracts
+
+from aio.functional import async_property
+
+from .base import GithubRepoEntity
+
+
+class AGithubTag(GithubRepoEntity, metaclass=abstracts.Abstraction):
+
+    @property
+    def commit_url(self):
+        """URL to retrieve related commit for this tag."""
+        # this is messy 8/
```

Comment:
as per [inclusive language policy](https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md#inclusive-language-policy) may be we can find some meaningful comment which explains the situation :)